### PR TITLE
Added Remove Covered Intervals LeetCode1288 CPP solution #hacktoberfest-PR

### PR DESCRIPTION
**Pull request by rosy-02**
- [x] Added a solution to Remove Covered Interval LeetCode Daily problem October challenge https://leetcode.com/explore/challenge/card/october-leetcoding-challenge/559/week-1-october-1st-october-7th/3483/

Would be grateful if you would accept my submission for the Leetcode solutions. 100% efficient using sort and count

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,5 +96,7 @@ Please add your name in the list in the following format:
 ` - Name - [File Name](File Address)`<br>
 Nikunj Aggarwal - [Binary-search_C++.cpp](Algorithms/Searching/Binary-search_C++.cpp)  
 [Priyank Rastogi](https://github.com/Priyankcoder/) - [standard-template-library](standard-template-library)
+[rosy-02](https://github.com/rosy-02) - [Remove covered intervals.cpp](https://github.com/cp-geeks-adgitm/Hacktoberfest/pull/66)
+
```
Comment (rosy-02) [inline README.md:100]: these are my changes

File: Leetcode-Solutions/Arrays/Remove covered intervals.cpp [hunk 2]
```diff
@@ -33,6 +33,6 @@ bool compare(vector<int>&A,vector<int>&B)
         }
     }
     return count;
-}
+}};
```
Comment (rosy-02) [inline Leetcode-Solutions/Arrays/Remove covered intervals.cpp:36]: added nracket and semicolon

**Review by rosy-02 — COMMENTED** [review]
submitted changes

**Comment by rosy-02** [comment]
Hey please merge my leetcode solution

**Comment by Nikunj-Aggarwal** [comment]
Please follow the naming conventions and your PR has some conflicts in readme commit, please look into it.

**Comment by cpgeeksadgitm** [comment]
Thanks for the contribution. Please contribute this year too 2022.

**Pull request merged**